src/docs/ftw-analytics/template-sitemap/index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Sitemap in Sharetribe Web Template
+slug: sitemap-in-template
+updated: 2023-10-12
+category: ftw-analytics
+ingress:
+  This guide describes how the default sitemap works in the Sharetribe
+  Web Template, and how to customise it.
+published: true
+---
+
+Your marketplace's sitemap helps search engines to process the correct
+information on your marketplace. The Sharetribe Web Template has
+built-in support for creating a sitemap (introduced in
+[this change set](https://github.com/sharetribe/web-template/pull/243)),
+and this article gives you some context into how sitemap generating
+works in the template.
+
+## Template sitemap structure
+
+By default, the template generates several files related to the sitemap.
+When running the template locally with _yarn run dev_, you can find
+these routes on the dev server port, i.e. localhost:3500.
+
+You can read more about the template sitemap and other resources in the
+[Resources README](https://github.com/sharetribe/web-template/blob/main/server/resources/README.md).
+
+### robots.txt
+
+The [your-domain]**/robots.txt** file details the authenticated routes
+that crawlers should avoid, and exposes the URL to the main sitemap. If
+you custom develop routes that require authentication, it is a good idea
+to specify them here.
+
+### sitemap-index.xml
+
+The [your-domain]**/sitemap-index.xml** file further exposes three
+separate sub-sitemaps. This is useful because it allows you more
+detailed tracking in analytics tools, and it reduces the size of each
+individual sitemap.
+
```

Comment:
```suggestion

This is the sitemap, you should give to [Google Search Console](https://www.sharetribe.com/help/en/articles/8413179-how-to-configure-google-search-console).

```

The link now leads to an article about how to configure the Google Search Console, but there might be a specific article written later on about how to feed the sitemap to Google Search Console.

Reply:
There is a section with an explanation and a link related to Google search console at the end of this article